app/moves/middleware.js

```diff
@@ -5,12 +5,19 @@ const {
   parseISO,
   isValid: isValidDate,
 } = require('date-fns')
-const { find, get } = require('lodash')
+const { find, get, chunk } = require('lodash')
 
 const moveService = require('../../common/services/move')
 
 const moveDateFormat = 'yyyy-MM-dd'
 
+const { MOVES_BATCH_SIZE } = require('../../config')
```

Comment:
Maybe move this up to line 11.

The pattern for files we've tried to use is:

- load external libraries
- new line
- load internal libraries
- new line
- define module constants

It just helps keep consistency and helps with knowing where to look for things within the files across the app.